AdminWebsite/AdminWebsite.AcceptanceTests/Steps/HearingDetailsSteps.cs

```diff
@@ -90,7 +91,7 @@ public void SelectQuestionnaireNotRequired()
         [Then(@"case type dropdown should not be populated")]
         public void ThenCaseTypeDropdownShouldNotBePopulated()
         {
-            switch (_scenarioContext.Get<string>("User"))
+            switch (_scenarioContext.Get<UserAccount>("User").Role)
```

Comment:
If _scenarioContext.Get<UserAccount>("User") is null, then .Role will throw an exception

Reply:
done for next commit